config/db_tools.standalone.sample.yaml

```diff
@@ -0,0 +1,63 @@
+# This configuration file is an example for standalone usage.
+#
+# The given parameters you will find in this file must not be set in a Symfony
+# application context, and will ignored if so.
+#
+# All other configuration you can find in the ./packages/db_tools.yaml file
+# can be added in this file, you must simply omit the 'db_tools:' top level
+# node.
+
+# Working directory is the path to which all relative file references will
+# be relative to. If none set, the path will be this file directory instead.
+workdir: /var/www/my_project/
+
+# Database connections.
+# One line per connection, a single database URL, all options as query
+# parameters. Connections will be made using makincorpus/query-builder
+# which will raise exceptions when invalid options are found.
+# There is less configuration amplitude than using doctrine/dbal in
+# Symfony, yet it should be enough in most case.
+# In case any options or specific behaviour is missing, please file
+# an issue at https://github.com/makinacorpus/php-query-builder/issues
+connections:
+    connection_one: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
+    connection_two: "mysql://username:password@hostname:port?version=8.1&other_option=..."
+
```

Comment:
is it possible to just write 

`connections: "pgsql://username:password@hostname:port?version=16.0&other_option=..."`

if i have only one to declare ?

Reply:
It's documented in the configuration reference page, it's also possible to write `connections: URL` directly. I can add it here as well.

Reply:
I'm going to add this here, but better documentation should come into #162.